editor/plugins/voxel_gi_editor_plugin.cpp

```diff
@@ -166,8 +166,10 @@ void VoxelGIEditorPlugin::_voxel_gi_save_path_and_bake(const String &p_path) {
 	probe_file->hide();
 	if (voxel_gi) {
 		voxel_gi->bake();
-		ERR_FAIL_COND(voxel_gi->get_probe_data().is_null());
-		ResourceSaver::save(voxel_gi->get_probe_data(), p_path, ResourceSaver::FLAG_CHANGE_PATH);
+		Ref<VoxelGIData> voxel_gi_data = voxel_gi->get_probe_data();
```

Comment:
Add a comment to explain why this is done:

```suggestion
		// Ensure the file is always saved to an external resource.
		// This avoids bloating the scene file with large binary data,
		// which would be serialized as Base64 if the scene is a `.tscn` file.
		Ref<VoxelGIData> voxel_gi_data = voxel_gi->get_probe_data();
```

Reply:
Done!